api/tasks_test.go

```diff
@@ -42,27 +40,26 @@ var createTaskPayloadFmt = `{
 	"number_of_documents": 5
 }`
 
-func expectedTask(jobID string, lastUpdated time.Time, numberOfDocuments int, taskName string) (models.Task, error) {
+func expectedTask(version, jobID, taskName string, lastUpdated time.Time, numberOfDocuments int) (models.Task, error) {
 	cfg, err := config.Get()
```

Comment:
**COMMENT 3**
You can simplify this by removing the `config.Get()` as this is already done within the test. You should instead include `bindAddr` parameter and use that value